docs/content/installation/nic-images/use-aws-image.md

```diff
@@ -78,12 +80,57 @@ Make sure you have an operational EKS cluster and that the namespace for your NG
         apiGroup: rbac.authorization.k8s.io
     ```
 
-3. Sign in to the AWS ECR registry that specified in the instructions on the [AWS Marketplace portal](https://aws.amazon.com/marketplace/pp/prodview-fx3faxl7zqeau?sr=0-1&ref_=beagle&applicationId=AWSMPContessa).
+1. Sign in to the AWS ECR registry that specified in the instructions on the [AWS Marketplace portal](https://aws.amazon.com/marketplace/pp/prodview-fx3faxl7zqeau?sr=0-1&ref_=beagle&applicationId=AWSMPContessa).
 
     {{< img title="ECR pull instructions for NGINX Ingress Controller" src="./img/ecr-pull-instructions.png" >}}
 
+1. Update the image in the _nginx-plus-ingress.yaml_ manifest.
```

Comment:
```suggestion
4. Update the image in the _nginx-plus-ingress.yaml_ manifest.
```

Reply:
Alan asked me to avoid using numbers, helm will figure out the correct numbers